sami2py/_core_class.py

```diff
@@ -291,28 +284,30 @@ def _load_model(self):
                                 'long_name': 'neutral wind velocity'})
 
         if self.MetaData['ExB model'] == 'Fourier Series':
```

Comment:
This should remain "ExB" right?

Reply:
Yes.  The ExB stuff is fixed downstream, you'll see it in the next pull.